xml/System.Globalization/CultureInfo.xml

```diff
@@ -2391,29 +2393,29 @@ Setting `predefinedOnly` to `true` will ensure a culture is created only if the
-  
- The RFC 4646 standard that is maintained by the Internet Engineering Task Force (IETF) defines an IETF language tag, which provides a uniform means of identifying a language. The format of an IETF language tag is the same as the culture name returned by the <xref:System.Globalization.CultureInfo.Name%2A> property, but does not identify a culture uniquely. Different cultures share the same IETF language tag if those cultures have identical linguistic characteristics. The linguistic characteristics of a culture are contained in the <xref:System.Globalization.TextInfo> object associated with a <xref:System.Globalization.CultureInfo> object.  
-  
- An IETF language tag consists of a mandatory language identifier, an optional script identifier, and an optional region identifier.  
-  
- Your applications should use the retrieved culture to control language-specific tasks, such as managing a spelling or grammar checker. Applications should not use the retrieved culture to control culture-specific tasks, such as sorting text or formatting times and dates.  
-  
- For more information, see RFC 4646, "Tags for Identifying Languages," and the most recent draft standard, Draft-phillips-langtags-05.txt, at [https://go.microsoft.com/fwlink/?LinkId=100231](https://go.microsoft.com/fwlink/?LinkId=100231).  
-  
+> This method and the <xref:System.Globalization.CultureInfo.IetfLanguageTag%2A> property are deprecated. Instead of using these elements, the application should use a new <xref:System.Globalization.CultureInfo> object and calls to the <xref:System.Globalization.CultureInfo.Name%2A> property. IETF tags and names are identical.
+
+ The RFC 4646 standard that is maintained by the Internet Engineering Task Force (IETF) defines an IETF language tag, which provides a uniform means of identifying a language. The format of an IETF language tag is the same as the culture name returned by the <xref:System.Globalization.CultureInfo.Name%2A> property, but does not identify a culture uniquely. Different cultures share the same IETF language tag if those cultures have identical linguistic characteristics. The linguistic characteristics of a culture are contained in the <xref:System.Globalization.TextInfo> object associated with a <xref:System.Globalization.CultureInfo> object.
+
+ An IETF language tag consists of a mandatory language identifier, an optional script identifier, and an optional region identifier.
+
+ Your applications should use the retrieved culture to control language-specific tasks, such as managing a spelling or grammar checker. Applications should not use the retrieved culture to control culture-specific tasks, such as sorting text or formatting times and dates.
+
+ For more information, see [RFC 4646 - "Tags for Identifying Languages](https://datatracker.ietf.org/doc/html/rfc4646).
```

Comment:
Should this be updated to RFC 5646?

Reply:
I wouldn't update that. The reason is GetCultureInfoByIetfLanguageTag is old and kind of obsolete. current CultureInfo can be created by BCP-47 tags which superseed IETF. I suggest adding a small comment here telling users normally should use the CultureInfo constructors or GetCultureInfo directly.